dataflux_pytorch/dataflux_mapstyle_dataset.py

```diff
@@ -165,7 +167,7 @@ def __len__(self):
     def __getitem__(self, idx):
         return self.data_format_fn(
             dataflux_core.download.download_single(
-                storage_client=self._get_or_create_storage_client(),
```

Comment:
I believe all references to this method have been removed, so we can remove the method too.